eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -101,34 +101,36 @@ if ($allPackageProperties)
     }
     foreach($pkg in $allPackageProperties)
     {
-        Write-Host "Package Name: $($pkg.Name)"
-        Write-Host "Package Version: $($pkg.Version)"
-        Write-Host "Package SDK Type: $($pkg.SdkType)"
-        Write-Host "Artifact Name: $($pkg.ArtifactName)"
-        Write-Host "Release date: $($pkg.ReleaseStatus)"
-        $configFilePrefix = $pkg.Name
-        if ($pkg.ArtifactName)
-        {
-          $configFilePrefix = $pkg.ArtifactName
+        if ($pkg.Name) {
```

Comment:
I think this is better fixed directly in the python repo at https://github.com/Azure/azure-sdk-for-python/blob/main/eng/scripts/Language-Settings.ps1#L27. In general we should never be getting invalid package entries back from Get-AllPkgProperties.

Reply:
I agree, but in my log example it _is being returned properly from that function_. You can see "Length 1"? That's what is being returned from the Language-Settings function. We're _somehow_ getting two additional outputs just on this platform, but NOT from that function.

Reply:
This is what I'm talkin bout: 
![image](https://github.com/Azure/azure-sdk-tools/assets/45376673/16f25971-756b-4241-a113-515c1b9b96c8)

That function _is_ returning the expected number, and then Get-AllPkgProperties is getting two additional before returning.

Reply:
"Returning PackageProps" is leaving the function that you're asking me to fix. "Length 1" is the lenght of PackageProps being returned.

Reply:
Very strange I cannot explain this behavior either. I'm looking through your code branch to try and understand how this could happen.